processes/release.md

```diff
@@ -1,3 +1,269 @@
+      - [Minor Version Bump (`release/**` -> `master` = `x.y.*` -> `x.y+1.0-next.0`)](#minor-version-bump-release---master--xy---xy10-next0)
+    - [Deployment checklist](#deployment-checklist)
+      - [Per $release](#per-release)
+      - [Per $chain](#per-chain)
+    - [Scripts](#scripts)
+      - [`cover-traffic` deployment script](#cover-traffic-deployment-script)
+      - [`topology` deployment script](#topology-deployment-script) -->
+
+## Types of release
+
+| Type     | Goal                                                  | Ambassadors | Public |
+| -------- | ----------------------------------------------------- | ----------- | ------ |
+| Internal | Test new features and bug fixes                       | MAYBE       | NO     |
+| Public   | New RPCh release, showcase new features and bug fixes | YES         | YES    |
+
+### Internal release
+
+- Internal releases are more frequent and their goal is to test new features and bug fixes.
+- There should be no public involvement unless the [internal release](#internal-release) is promoted to a [public release](#public-release).
+- All releases start as an [internal release](#internal-release) and may be promoted to a [public release](#public-release), see [release promotion](#release-promotion).
+- When an internal release is created, one of the [representatives](./development.md#representatives) is assigned to oversee the release cycle, this includes:
+  - notifies the HOPR team that the [internal release](#internal-release) has been created in element channel `releases`
+  - [creating release](#release-cycle)
+  - [testing release](#testing-phases)
+  - [promoting release](#release-promotion)
+
+### Public release
+
+- Public releases are less frequent and their goal is to showcase new features and bug fixes.
+- A [public release](#public-release) occurs when an [internal release](#internal-release) is promoted to a [public release](#public-release), see [release promotion](#release-promotion).
+
+## When to release
+
+### Milestone based releases
+
+At the end of a sprint, if a sufficient amount of features / bug fixes were implemented since last release, the [Representatives](./development.md#representatives) and `founders` may queue and prioritize a new release for the **upcoming** sprint, this happens during [epic prioritization](./development.md#epic-prioritization).
+This new release is considered an [internal release](#internal-release) and may be [promoted](#release-promotion) to a [public release](#public-release) if [testing phases](#testing-phases) are successful.
+
+### Deadline based releases
+
+Deadline based releases are releases which have a defined deadline and scope agreed between the [Representatives](./development.md#representatives) and `founders`.
+These releases should be of a large scope and occur only on occasions where a release is necessary.
```

Comment:
Is this release going to go through the same flow as the milestone release:`internal -> testing -> public` or is it going straight to public?

Reply:
Same